ocs_ci/deployment/aws.py

```diff
@@ -79,6 +79,18 @@ def add_node(self):
         # TODO: Implement later
         super(AWSBase, self).add_node()
 
+    def check_cluster_existence(self):
+        """
+        Check cluster existence
+
+        """
+        cluster_name = config.ENV_DATA['cluster_name']
+        pattern = cluster_name.split("-")[0] + '*'
+        if self.aws.get_instances_by_name_pattern(pattern):
+            logger.warning(f"Cluster with name {cluster_name} already exists")
```

Comment:
This sentence might not be true, because you are checking only for the cluster name prefix and here you will report the specified _cluster_name_, not the name of the really existing cluster...

E.g. I have previously installed cluster named `dahorak-foo` and I would like to install new cluster `dahorak-bar`. But it will report:
```
WARNING: Cluster with name dahorak-bar already exists
```